### ISSUE TITLE
Undefined $file['tmp_name']

### ISSUE THREAD

**Issue by aksld** [darylldoyle/safe-svg#45]
### Describe the bug

When I submit a form from another extension, there is an empty file entry.
I have an error generated by safe-svg because there is an undefined variable.

In function `check_for_svg` :

`$wp_filetype = wp_check_filetype_and_ext( $file['tmp_name'], $file['name'] );`

If I add this code before, it's working !

```
if ( ! isset( $file['tmp_name'] ) || ! isset( $file['name'] ) ) {
    return $file;
}
```

### Steps to Reproduce

Maybe create a form with empty file input.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dkotter** [comment]
@aksld Thanks for the report! I wasn't able to reproduce this scenario myself but I can see why the issue would happen. I've added a fix for this in #46, wondering if you have some time to test that out and see if that fixes your issue?

**Comment by aksld** [comment]
Perfect!
The fix remove the php error.

Thanks :)
